addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -126,9 +127,15 @@
                     <meta itemprop="name" t-att-content="job.company_id.name"/>
                     <meta itemprop="logo" t-attf-content="/logo.png?company=#{job.company_id.id}"/>
                 </span>
-                <span itemprop="jobLocation" itemscope="itemscope" itemtype="https://schema.org/Place">
-                    <meta itemprop="address" t-att-content="job.address_id.sudo().contact_address"/>
+                <span t-if="job_location" itemprop="jobLocation" itemscope="itemscope" itemtype="https://schema.org/Place">
+                    <meta itemprop="address" t-att-content="job_location"/>
```

Comment:
Changing the value of `t-att-content` should not be done in a stable version, so please don't change it and use the same value for the `t-if`. Therefore, the `job_location` variable will no longer be needed.

Reply:
Thank you for letting me know! I have changed the `t-att-content` back to it's original value and use the same value for the `t-if`. Please let me know if there is any part I will need to concern with 😉